.github/workflows/torch-inductor/scripts/install_torchinductor.sh

```diff
@@ -2,19 +2,24 @@
 
 # remember where we started
 ROOT="$(pwd)"
+MODEL_SPEC=$1
 
 # torchinductor venv
 whoami
+# clean up old venv
+rm -rf /opt/torchinductor_venv
 python3 -m venv /opt/torchinductor_venv
 # shellcheck source=/dev/null
 source /opt/torchinductor_venv/bin/activate
 # shellcheck source=/dev/null
-source ./.github/workflows/torchinductor/scripts/common.sh
+source ./.github/workflows/torch-inductor/scripts/common.sh
 
 # pytorch nightly
-pip3 install --force-reinstall --pre torch torchtext torchvision torchaudio --extra-index-url https://download.pytorch.org/whl/nightly/cu118
+pip3 install --force-reinstall --pre torch torchtext torchvision torchaudio torchrec --extra-index-url https://download.pytorch.org/whl/nightly/cu121
```

Comment:
We should pin a known good version here because looking at the pytorch benchmark CI, the latest pytorch nightlies often have spurious failures that aren't valuable for us to catch.

Reply:
hmm, that means we need to update the CI manually. I don't know if that's a good idea

Reply:
We could read them from a single file we update? But in general, I don't think we will have to advance it super  frequently.